packages/lit-dev-content/site/css/blog-post.css

```diff
@@ -1,92 +1,98 @@
 @import url('./blog-common.css');
 
-#preamble {
-  font-size: 1.18em;
-  border-bottom: 1.5px solid var(--color-cyan);
-  padding-bottom: 1em;
+h1 {
+  margin-bottom: 12px;
 }
 
-#frontMatter a {
-  color: white;
+#summary {
+  font-style: italic;
 }
-#frontMatter a:hover {
-  text-decoration: underline;
+
+#date {
+  color: #666;
 }
 
-#imageNameDate {
+.author {
   display: flex;
-  margin-top: 1.5em;
-  align-items: center;
+  margin: 0 32px;
+  border-bottom: 1.5px solid var(--color-blue);
+  padding: 0 0 32px 0;
+  margin: 32px 0;
 }
 
-#imageNameDate > img {
-  width: 3em;
-  height: 3em;
-  margin-right: 1em;
+.author > img {
+  width: 70px;
+  height: 70px;
+  border-radius: 50%;
 }
 
-#frontMatter > h1 > img {
-  vertical-align: middle;
-  margin-right: 0.5em;
+.authorRhs {
+  display: flex;
+  flex-direction: column;
+  margin-left: 32px;
+  justify-content: space-around;
 }
 
-#frontMatter > h1 > img {
-  width: 1.2em;
-  height: 1.2em;
-  background: white;
-  padding: 0.1em;
+cite.authorName {
+  font-style: normal;
+  color: #222;
+  font-weight: 500;
 }
 
-#nameDate > * {
+ul.authorLinks {
+  padding-left: 0;
   margin: 0;
 }
 
-#authorName {
-  font-weight: 600;
-  font-size: 1.1em;
+ul.authorLinks > li {
+  display: inline-block;
+  text-indent: 0;
 }
 
-#blogContent {
-  padding: var(--blog-top-padding) 3em;
-  font-family: "Open Sans", sans-serif;
-  font-size: 0.88889em;
+ul.authorLinks > li:not(:first-of-type)::before {
+  content: "·";
+  margin: 0 8px 0 4px;
```

Comment:
SUPER NIT: Interested why not center the dot? I.e. `margin: 0 8px;`?

Reply:
Hmm weird. On Linux it is centered currently, but you're right on macOS it is off-center.

Reply:
Turns out it was a dev mode vs production mode difference! Very weird, I truly could not figure out why they were rendering differently though -- all relevant CSS seemed identical. Using flex has made it consistent.